<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-VHO7M4VD" failures="2" tests="2" name="Default test(failed)" time="11.024" errors="0" timestamp="2022-11-04T10:30:23 IST">
  <testcase classname="TestCases.FailedTest" name="verifyTitleTest" time="0.046">
    <failure type="java.lang.AssertionError" message="expected [1Swag Labs] but found [Swag Labs]">
      <![CDATA[java.lang.AssertionError: expected [1Swag Labs] but found [Swag Labs]
at TestCases.FailedTest.verifyTitleTest(FailedTest.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitleTest -->
  <testcase classname="TestCases.FailedTest" name="verifyBotlogoTest" time="0.122">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FailedTest.verifyBotlogoTest(FailedTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyBotlogoTest -->
  <testcase classname="TestCases.FailedTest" name="loginToApp11Test" time="0.0">
    <ignored/>
  </testcase> <!-- loginToApp11Test -->
  <testcase classname="TestCases.FailedTest" name="loginToApp2Test" time="0.0">
    <ignored/>
  </testcase> <!-- loginToApp2Test -->
  <testcase classname="TestCases.FailedTest" name="verifyUrlTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUrlTest -->
</testsuite> <!-- Default test(failed) -->
